src/UI/Areas/Identity/Pages/Account/Manage/TwoFactorAuthentication.cshtml

```diff
@@ -29,6 +29,12 @@
         </div>
     }
 
+    if (Model.IsMachineRemembered)
+    {
+        <form method="post" style="display: inline-block">
```

Comment:
REVIEW: Is this the 'right' way to get the buttons on the same row?

Reply:
This is fine. Bootstrap 4.0 has a utility class for this, but I guess we're on an older version.